corehq/apps/locations/bulk_management.py

```diff
@@ -403,14 +382,19 @@ def validate_and_parse_stubs_from_excel(self):
                             missing=", ".join(expected - actual))
                     )
                 location_stubs.extend(self._get_locations(sheet_reader, sheet_name))
-        return type_stubs, location_stubs
-
-    def _get_types(self, rows):
-        # takes raw excel row dicts and converts them to list of LocationTypeStub objects
-        return [
-            LocationTypeStub.from_excel_row(row, index)
-            for index, row in enumerate(rows)
-        ]
+        return type_data, location_stubs
+
+    @staticmethod
+    def _get_type_data(index, row):
+        titles = LOCATION_TYPE_SHEET_HEADERS
+        name = row.get(titles['name'])
```

Comment:
nit: `row.get(titles['name'], '').strip()` would help ensure empty spaces are removed.

Reply:
Huh, it looks like this could use a fair bit more cleaning - I don't see anything that makes sure `site_codes` are valid slugs, either.  Thanks for flagging.